src/Makefile.in

```diff
@@ -291,6 +292,9 @@ mdb-support/libmtev.so:	mdb-support/libmtev.lo mdb-support/mdb_help.lo
 	@echo "- linking $@"
 	$(Q)$(MODULELD) $(SHLDFLAGS) -o $@ mdb-support/libmtev.lo mdb-support/mdb_help.lo
 
+aklomp-base64/lib/libbase64.o:
+	AVX2_CFLAGS=-mavx2 SSSE3_CFLAGS=-mssse3 SSE41_CFLAGS=-msse4.1 SSE42_CFLAGS=-msse4.2 AVX_CFLAGS=-mavx $(MAKE) -C aklomp-base64 lib/libbase64.o
```

Comment:
Does this mean that if we happen to build on a system that lacks AVX2, it won't be available at runtime?

Reply:
No. All variants are unconditionally built. Selected at runtime.